index.js

```diff
@@ -22,14 +23,21 @@ function decode (uri) {
 
 function encode (address, options) {
   options = options || {}
+
+  var scheme = 'bitcoin'
+  if (options.scheme) {
+    scheme = options.scheme
+    delete options.scheme
```

Comment:
FWIW, this would modify the `options` object... bad practice.

Reply:
You are right, I will fix this!